lib/puma/cluster.rb

```diff
@@ -211,12 +211,11 @@ def worker(index, master)
         pipes[:wakeup] = @wakeup
       end
 
-      server = start_server if preload?
```

Comment:
I'm pretty sure the only reason we start the server here and not in the new worker is to [ensure the loaded cluster app is used during server initialization](https://github.com/puma/puma/blob/edd2a1728e39d055545fd12ffdfa42d5d4ccbc58/lib/puma/runner.rb#L184). Now that we can pass that app into the worker, we can consistently start the server there instead.